File: pophory-domain/src/main/java/com/pophory/pophorydomain/studio/infrastructure/StudioJpaRepository.java
package com.pophory.pophorydomain.studio.infrastructure;


import com.pophory.pophorydomain.studio.Studio;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface StudioJpaRepository extends JpaRepository<Studio, Long> {

    Optional<Studio> findByName(String name);
}

File: pophory-domain/src/main/kotlin/com/pophory/pophorydomain/studio/infrastructure/StudioJpaRepository.kt
package com.pophory.pophorydomain.studio.infrastructure

import com.pophory.pophorydomain.studio.Studio
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface StudioJpaRepository : JpaRepository<Studio, Long> {
    fun findByName(name: String?): Optional<Studio>
}